server.js

```diff
@@ -102,6 +219,3 @@ function Movies(movies) {
   this.image_url = movies.poster_path;
```

Comment:
missing 'https://image.tmdb.org/t/p/w200_and_h300_bestv2' + before the image path.